README.md

```diff
@@ -59,6 +63,9 @@ docker build -t snowplow/s3-loader:0.6.0 s3-loader/0.6.0
 
 # Iglu Server image
 docker build -t snowplow/iglu-server:0.3.0 iglu-server/0.3.0
+
+# Piinguin Server image
+docker build -t snowplow/piinguin-server:0.1.0 piinguin-server/0.3.0
```

Comment:
Shouldn't this be `piinguin-server/0.1.0` not `piinguin-server/0.3.0`?

Reply:
yes